Packages/Sites/Sfi.Kateheo/Resources/Private/Partials/2levelMenu.html

```diff
@@ -0,0 +1,18 @@
+{namespace neos=TYPO3\Neos\ViewHelpers}
+{namespace ts=TYPO3\TypoScript\ViewHelpers}
+<ul{attributes -> f:format.raw()}>
+	<f:render section="itemsList" arguments="{items: items}" />
+</ul>
+
+<f:section name="itemsList">
+	<f:for each="{items}" as="item">
+		<li{ts:render(path: '{item.state}.attributes', context: {item: item}) -> f:format.raw()}>
+		<neos:link.node node="{item.node}" />
+		<f:if condition="{item.subItems}">
+			<ul class="nav nav-list bs-docs-sidenav affix-top">
```

Comment:
You hardcode nested menu classes in the template, while you set top menu classes in TS. It's not cool.
Better add another property like `<ul {secondLevelAttributes -> f:format.raw()}>` and set the classes in TS.